src/renderer/react/price-box/containers/price-box.js

```diff
@@ -1,6 +1,6 @@
 import { connect } from 'react-redux';
 import { CryptoPriceBox } from 'selfkey-ui';
-import { getLocale } from 'common/locale/selectors';
+import { getLocale } from 'locale/selectors';
```

Comment:
@sk91 here is the same. I think we should keep what is common between process on the common directory. Thoughts?